product_docs/docs/postgres_distributed_for_kubernetes/1/backup.mdx

```diff
@@ -71,32 +71,32 @@ spec:
       immediate: true
 ```
 
-Scheduled Backups can be suspended if necessary by setting `.spec.backup.cron.suspend` to true. This will
-prevent any new backup from being scheduled while the option is set to true.
+You can suspend scheduled backups by setting `.spec.backup.cron.suspend` to `true`. This setting
+prevents any new backup from being scheduled.
 
-In case you want to execute a backup as soon as the ScheduledBackup resource is created
-you can set `.spec.backup.cron.immediate` to true.
+If you want to execute a backup as soon as the `ScheduledBackup` resource is created,
+you can set `.spec.backup.cron.immediate` to `true`.
```

Comment:
Is ScheduledBackup supposed to be one word? is this really a resource name or a type of resource? I'm wondering if this should just be: the scheduled-backup resource.